services/apps/alcs/src/alcs/application-decision/application-decision-v2/application-decision-v2.module.ts

```diff
@@ -76,6 +77,7 @@ import { NaruSubtype } from '../../../portal/application-submission/naru-subtype
   controllers: [
     ApplicationDecisionV2Controller,
     ApplicationDecisionComponentController,
+    ApplicationDecisionConditionController,
```

Comment:
Do we need to mock this in test or is the `.remove` basically it?

Reply:
Nope, this is a module and you do not need to mock a module in a test. Only injectables that are used in the service/controller need to be mocked. Also, you've highlighted the controller where .remove is for the service test 🙂